docs/UserGuide.adoc

```diff
@@ -37,6 +37,7 @@ What's different from AddressBook-Level1:
 * Support for storing address (`a/`) and tags (`t/`)
 * Support for marking a contact detail as 'private' (`pa/`) (`pe/`) (`pp/`)
 * View details of a person (`view` : shows non-private details), (`viewall` : shows all details)
+* Sort list of person in address book (`sort`)
```

Comment:
That's good that you have updated the User Guide (UG) to show the newly implemented function that you have created. The UG is the place where the user will go to find out the different features and the usage of it most of the time.